adapter/service/waterheater/service.go

```diff
@@ -347,6 +347,7 @@ func (s *service) supportedRange(mode string) *Range {
 
 	supportedRange = &Range{}
 	ok = s.Service.Specification().PropertyObject(PropertySupportedRange, supportedRange)
+
```

Comment:
I guess that empty line is not needed. Probably better to add one before assigning to `ok`